sklearn/svm/src/liblinear/linear.cpp

```diff
@@ -1706,10 +1734,10 @@ static int solve_l1r_l2_svc(
 // solution will be put in w
 //
 // See Yuan et al. (2011) and appendix of LIBLINEAR paper, Fan et al. (2008)
-#undef SAMPLE_WEIGHT
-#define SAMPLE_WEIGHT(i) C[y[i]+1]*sample_weight[i]
-// To support weights for instances, use SAMPLE_WEIGHT(i)
-// Each instance is weighted by (class_weight*sample_weight)
+
+#undef GETI
+#define GETI(i) (i)
```

Comment:
we don't need these `GETI`s now, do we?

Reply:
yes we do. It is to replace `SAMPLE_WEIGHT(i)` to synchronize with the original code instead of our custom version.